### ISSUE TITLE
error message from accessing a missing property of a node returned from get_node() incorrectly says get_node() returned 'Nil'

### ISSUE THREAD

**Issue by RefinedSoftwareLLC** [godotengine/godot#1196]
error message from accessing a missing property of a node returned from get_node() incorrectly says get_node() returned 'Nil' when it did return the node correctly.

add a Button node to the scene (named Button).
set a script for the room with this code:

```
func _ready():
	print(get_node("Button"))
	print(get_node("Button").misspelled)
```

and see how the output is different then this:

```
func _ready():
	print(get_node("Button"))
	var a = get_node("Button")
	print(a.misspelled)
```

get_node() is not returning nil like the first error message says, it is returning the Button node correctly. Both error messages should be exactly the same.

**Comment by akien-mga** [comment]
Can't reproduce in the current master, so it has likely been fixed, e.g.:
```print(get_node("/root/global").widthk)```
Properly gives this error message:
```Invalid get index 'widthk' (on base: 'Node (global.gd)').```

**Comment by akien-mga** [comment]
No actually I can still reproduce it if I try to access a Button node like in the original report, e.g.:
```
	print(get_node("MainMenu/NewGame"))
	print(get_node("MainMenu/NewGame").get_paren())
```
The first print correctly gives the RID of the Button node, but the second print gives:
```
Invalid call. Nonexistent function 'get_paren' in base 'Nil'.
```

**Comment by akien-mga** [comment]
The error string is built here: https://github.com/godotengine/godot/blob/07e9741/modules/gdscript/gd_script.cpp#L712
So somehow the node is not seen as a Variant::OBJECT here: https://github.com/godotengine/godot/blob/07e9741/modules/gdscript/gd_script.cpp#L158, but I'm not familiar enough with the Variant format to say if it's a bug or the expected behaviour.

**Comment by akien-mga** [comment]
@reduz As you're improving the debugger/adding a profiler, it would be nice to get a fix for this one.

**Comment by akien-mga** [comment]
Not critical for the upcoming 2.1, so moving to the next milestone.

**Comment by vnen** [comment]
This was fixed by 27cb751 (and 3754f6c for functions).
